target/test-classes/features/BookDepository.feature

```diff
@@ -0,0 +1,35 @@
+@Parallel
```

Comment:
I've verified that parallelization works. However, if we want to see that it works in parallel with the current configuration - another feature file is required. All those scenarios are being executed one by one for this feature file